src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -104,34 +109,45 @@ public boolean extractQuestionDetails(
         return true;
     }
 
-    private List<Double> getRubricWeights(Map<String, String[]> requestParameters, int numOfRubricChoices,
-                                         boolean hasAssignedWeights) {
-        List<Double> rubricWeights = new ArrayList<>();
+    private List<List<Double>> getRubricWeightsForEachCell(Map<String, String[]> requestParameters,
```

Comment:
Maybe add a comment explain how this method is different from `getRubricWeights`? Since this method's name may confuse others that it is the getter for `rubricWeightsForEachCell`.